x-pack/plugins/index_lifecycle_management/__jest__/policy_table.test.tsx

```diff
@@ -52,6 +52,8 @@ const testPolicy = {
   },
 };
 
+const isLinkedWithIndices = (i: number) => i > 0 && i % 2 === 0;
```

Comment:
Also, how about adjusting the test for this behavior on line 236, so that it consumes the helper?

https://github.com/elastic/kibana/blob/6ccf6e14ee20c1ab3d4e8461b865ca70f2ed2016/x-pack/plugins/index_lifecycle_management/__jest__/policy_table.test.tsx#L236

```js
  test('delete policy button is enabled when there are no linked indices', () => {
    const rendered = mountWithIntl(component);
    const visiblePolicies = getPolicies(rendered);
    const usedPolicy = visiblePolicies.find((p) => p. isUsedByAnIndex);
    expect(usedPolicy).toBeDefined();

    const policyRow = findTestSubject(rendered, `policyTableRow-${usedPolicy!.name}`);
    const deleteButton = findTestSubject(policyRow, 'deletePolicy');
    expect(deleteButton.props().disabled).toBeFalsy();
  });
```

Reply:
Updated here https://github.com/elastic/kibana/pull/132269/commits/e36d273a64c92baed0cde1a62d933c057ad3c8a2